src/writer.js

```diff
@@ -14,13 +14,15 @@ function hasFiles (configuration) {
 function ensurePath (dir, fullPath) {
   return new Promise((resolve, reject) => {
     if (!fs.existsSync(dir)) {
-      mkdirp(dir, (err) => {
-        if (err) {
-          reject(new Error(`Failed to create path '${dir}' for configuration file '${fullPath}':\n\t${err.message}`))
-        } else {
-          resolve()
-        }
-      })
+      mkdirp(dir).then(
+        () => { resolve() },
+        (err) => {
+          if (err) {
+            reject(new Error(`Failed to create path '${dir}' for configuration file '${fullPath}':\n\t${err.message}`))
+          } else {
```

Comment:
You probably don't need that `else` block anymore? But fine to keep it if you think it's safer this way. 🤔

Reply:
Keeping it in :)